<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareToolWidget</class>
 <widget class="QWidget" name="CompareToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompareToolWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_Left">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_Left">
       <item>
        <widget class="QComboBox" name="comboBox_Left"/>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_Left">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Left">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeView_Left"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_Right">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_Right">
       <item>
        <widget class="QComboBox" name="comboBox_Right"/>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_Right">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Right">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeView_Right"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
